jaas/src/main/java/com/griddynamics/jagger/jaas/storage/ProjectDao.java

```diff
@@ -0,0 +1,58 @@
+package com.griddynamics.jagger.jaas.storage;
+
+import com.griddynamics.jagger.jaas.storage.model.ProjectEntity;
+import org.hibernate.SessionFactory;
+import org.hibernate.classic.Session;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Repository;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+
+@SuppressWarnings("unchecked")
+@Repository
+public class ProjectDao implements ProjectEntityDao {
```

Comment:
It's better to follow such naming convention: MyDao - interface, MyDaoImpl - implementation, or smth like this.
In case of ProjectDao and ProjectEntityDao it's unclear which is interface and which is implementation.